lib/octokit/client/deployments.rb

```diff
@@ -1,69 +1,87 @@
 module Octokit
   class Client
-
     # Methods for the Deployments API
     #
-    # @see https://developer.github.com/v3/repos/commits/deployments/
+    # @see https://developer.github.com/v3/repos/deployments/
     module Deployments
-
-      # Fetch a single deployment for a repository
+      # Get a single deployment
```

Comment:
The new documentation comes directly from the route file, which is generated from the docs site. I'd prefer to be consistent with that documentation, even if it's slightly less descriptive than the old stuff.

Reply:
I think the docs team is also happy to accept small pull requests that improve the descriptions, this way all the Octokit REST clients will benefit, too :)